src/olympia/amo/indexers.py

```diff
@@ -165,20 +165,21 @@ def extract_field_search_translation(cls, obj, field, default_locale):
     @classmethod
     def extract_field_analyzed_translations(cls, obj, field, db_field=None):
         """
-        Returns a dict containing translations for each language-specific
-        analyzer for the given field. Empty translations are skipped entirely.
+        Returns a dict containing translations for each language that we have
+        an analyzer for, for the given field.
+
+        When no translation exist for a given language+field combo, the value
+        returned is an empty string, to avoid storing the word "None" as the
+        field does not understand null values.
```

Comment:
is just not including the language+field combination in the dict not an option?

Reply:
It's an option, but I went with what I thought was the simplest option that stayed close to the previous behavior (previously, we would store an empty list, because we potentially stored multiple values, now we store an empty string)